source/ulid.ts

```diff
@@ -100,11 +100,11 @@ function detectRoot(): any {
 }
 
 export function encodeRandom(len: number, prng: PRNG): string {
-    let str = "";
-    for (; len > 0; len--) {
-        str = randomChar(prng) + str;
```

Comment:
`randomChar` could be removed if no longer used.

Reply:
As I reverted back to the previous implementation, I kept it.